code/modules/reagents/chemistry/holder.dm

```diff
@@ -1436,12 +1436,15 @@
  * - delta_energy: The amount to change the thermal energy by.
  * - min_temp: The minimum temperature that can be reached.
  * - max_temp: The maximum temperature that can be reached.
+ * - old_heat_capacity: Heat capacity before thermal energy change if they are part of the reaction.
  */
-/datum/reagents/proc/adjust_thermal_energy(delta_energy, min_temp = 2.7, max_temp = 1000)
+/datum/reagents/proc/adjust_thermal_energy(delta_energy, min_temp = 2.7, max_temp = 1000, old_heat_capacity)
```

Comment:
```suggestion
/datum/reagents/proc/adjust_thermal_energy(delta_energy, min_temp = TCMB, max_temp = CHEMICAL_MAXIMUM_TEMPERATURE, old_heat_capacity)
```

Reply:
I hope nothing relies on the fact that adjust_thermal_energy default max_temp parameter is some random ass number.